src/compiler/checker.ts

```diff
@@ -11707,7 +11707,7 @@ module ts {
                 return getEnumMemberValue(<EnumMember>node);
             }
 
-            let symbol = getNodeLinks(node).resolvedSymbol;
+            let symbol = getSymbolAtLocation(node);

```

Comment:
`getSymbolAtLocation` is a top language service API that should not be used on the checker itself. For this particular case I think you should use `getResolvedSymbol`. Also implementation of `getResolvedSymbol` should be augmented (i.e. by adding an extra argument) so in this case it won't report an error when name cannot be resolved